operator/src/main/resources/scripts/utils.sh

```diff
@@ -889,3 +889,81 @@ function adjustPath() {
     fi
   fi
 }
+
+#
+# checkCommonMount
+#   purpose: If the COMMON_MOUNT_PATH directory exists, it echoes the contents of output files
+#            in ${COMMON_MOUNT_PATH}/commonMountLogs dir. It returns 1 if a SEVERE message
+#            is found in any of the output files in ${COMMON_MOUNT_PATH}/commonMountLogs dirs.
+#            It also returns 1 if 'successfully' message is not found in the output files
+#            or if the COMMON_MOUNT_PATH directory is empty. Otherwise it returns 0 (success).
+#            See also 'commonMount.sh'.
+#
+function checkCommonMount() {
+  # check commonMount results (if any)
+  if [ ! -z "$COMMON_MOUNT_PATH" ]; then
+    trace FINE "Common Mount: COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH'."
+    traceDirs $COMMON_MOUNT_PATH
+    touch ${COMMON_MOUNT_PATH}/testaccess.tmp
+    if [ $? -ne 0 ]; then
+      trace SEVERE "Common Mount: Cannot write to the COMMON_MOUNT_PATH '${COMMON_MOUNT_PATH}'" && return 1
```

Comment:
minor: Since COMMON_MOUNT_PATH is an internal var name from an admin/end-user perspective, consider adding ". This path is configurable using the domain resource 'serverPod.commonMount.mountPath' attribute."